source/_integrations/velbus.markdown

```diff
@@ -60,15 +60,17 @@ You can use the `velbus.sync_clock` action to synchronize the clock of the Velbu
 
 | Data attribute | Optional | Description                              |
 | ---------------------- | -------- | ---------------------------------------- |
-| `interface`            | no       | The port used to connect to the bus (the same one as used during configuration). |
+| `interface`            | no       | The port used to connect to the bus (the same one as used during configuration), this is deprecated on 2025.2 and will be removed in 2025.8. |
```

Comment:
we don't actually list the version of when it is deprecated. we usually remove the docs when it is deprecated.